lib/Analyses/DynamicAliasAnalysis.cpp

```diff
@@ -167,6 +167,10 @@ void DynamicAliasAnalysis::removePointedBy(Definition Ptr, Location Loc) {
   auto J = PointedBy.find(Loc);
   assert(J != PointedBy.end());
   J->second.erase(Ptr);
+  /*Do not keep those Location entry which does not map to any Defintion set*/
```

Comment:
I haven't played with pull requests very often. Update the current pull request or create another one.